### PR TITLE
Trouble shooting cleanup

### PR DESCRIPTION
**Pull request by keichwa**
starting...

### CODE REVIEW

File: modules/administration/pages/tshoot-packages.adoc [hunk 1]
```diff
@@ -30,11 +30,12 @@ When you have located the problematic process, the metadata can be regenerated a
...
 . Restart taskomatic:
 +
+
 ----
-/etc/init.d/taskomatic restart
+service taskomatic restart
 ----
 +
-. Check package locks and exclude lists to determine if packages are locked or excluded on the client:
+. On the client, Check package locks and exclude lists to determine if packages are locked or excluded:
```
Comment (Loquacity) [inline modules/administration/pages/tshoot-packages.adoc]:
```suggestion
. On the client, check package locks and exclude lists to determine if packages are locked or excluded:
```

File: modules/administration/pages/tshoot-rpctimeout.adoc [hunk 2]
```diff
@@ -27,30 +27,30 @@ While this will not resolve networking problems, it will cause a process to fail
 
 .Procedure: Resolving RPC connection timeouts
 
-. On the {productname} Server, open the [filename]``/etc/rhn/rhn.conf`` file and set a maximum timeout value (in seconds) for this parameter:
+. On the {productname} server, open the [filename]``/etc/rhn/rhn.conf`` file and set a maximum timeout value (in seconds):
```
Comment (Loquacity) [inline modules/administration/pages/tshoot-rpctimeout.adoc]: Shouldn't Server/Proxy be capped if it comes after {productname}?

Comment (keichwa) [reply modules/administration/pages/tshoot-rpctimeout.adoc]: Sounds like a clear rule ;)

File: modules/administration/pages/tshoot-intro.adoc [hunk 3]
```diff
@@ -14,21 +14,20 @@ These reports can be helpful for taking inventory of your entitlements, subscrib
...
-. Entitlements: lists all organizations on {susemgr}, sorted by system or channel entitlements.
+. System Inventory: lists all of the systems registered to {productname}.
+. Entitlements: lists all organizations on {productname}, sorted by system or channel entitlements.
 . Patches: lists all the patches relevant to the registered systems and sorts patches by severity, as well as the systems that apply to a particular patch.
-. Users: lists all the users registered to {susemgr} and any systems associated with a particular user.
+. Users: lists all the users registered to {productname} and any systems associated with a particular user.
 
-[command]``spacewalk-report`` allows administrators to organize and display reports about content, systems, and user resources across {susemgr}.
-To get the report in CSV format, run the following at the command line of your {susemgr} server.
+To get the report in CSV format, run the following at the command line of your {productname} server.
```
Comment (keichwa) [inline modules/administration/pages/tshoot-intro.adoc]:
```suggestion
To get the report in CSV format, run the following at the command line of your {productname} Server.
```

File: modules/administration/pages/tshoot-osadjabberd.adoc [hunk 4]
```diff
@@ -81,19 +112,59 @@ If you are experiencing bugs regarding OSAD, it can be useful to dump network me
...
 
 .Procedure: Server Side Capture
-. Install the [package]#tcpdump# package on the SUSE Manager Server as root: [command]``zypper in tcpdump ``
-. Stop the OSA dispatcher and Jabber processes with [command]``rcosa-dispatcher stop`` and [command]``rcjabberd stop.``
-. Start data capture on port 5222: [command]``tcpdump -s 0 port 5222 -w server_dump.pcap``
-. Start the OSA dispatcher and Jabber processes: [command]``rcosa-dispatcher start`` and [command]``rcjabberd start``.
-. Open a second terminal and execute the following commands: [command]``rcosa-dispatcher start`` and [command]``rcjabberd start``.
-. Operate the SUSE Manager server and clients so the bug you formerly experienced is reproduced.
-. Once you have finished your capture re-open terminal 1 and stop the capture of data with: kbd:[CTRL+c]
+. Install the [package]#tcpdump# package on the {productname} server as root:
```
Comment (keichwa) [inline modules/administration/pages/tshoot-osadjabberd.adoc]:
```suggestion
. Install the [package]#tcpdump# package on the server as root:
```

File: modules/administration/pages/tshoot-osadjabberd.adoc [hunk 5]
```diff
@@ -81,19 +112,59 @@ If you are experiencing bugs regarding OSAD, it can be useful to dump network me
...
+tcpdump -s 0 port 5222 -w server_dump.pcap
+----
+. Open a second terminal and start the OSA dispatcher and Jabber processes:
++
+
+----
+rcosa-dispatcher start
+rcjabberd start
+----
+. Operate the {productname} server and clients so the bug you formerly experienced is reproduced.
```
Comment (keichwa) [inline modules/administration/pages/tshoot-osadjabberd.adoc]:
```suggestion
. Operate the server and clients so the bug you formerly experienced is reproduced.
```

File: modules/administration/pages/tshoot-osadjabberd.adoc [hunk 6]
```diff
@@ -81,19 +112,59 @@ If you are experiencing bugs regarding OSAD, it can be useful to dump network me
...
+----
+tcpdump -s 0 port 5222 -w client_client_dump.pcap
+----
+. Open a second terminal and start the OSA process:
++
+
+----
+rcosad start
+----
+. Operate the {productname} server and clients so the bug you formerly experienced is reproduced.
```
Comment (keichwa) [inline modules/administration/pages/tshoot-osadjabberd.adoc]:
```suggestion
. Operate the server and clients so the bug you formerly experienced is reproduced.
```

File: modules/administration/pages/tshoot-rpctimeout.adoc (hunk 2 above)
Comment (keichwa) [inline modules/administration/pages/tshoot-rpctimeout.adoc]:
```suggestion
. On the {productname} Server, open the [filename]``/etc/rhn/rhn.conf`` file and set a maximum timeout value (in seconds):
```

File: modules/administration/pages/tshoot-rpctimeout.adoc [hunk 7]
```diff
@@ -27,30 +27,30 @@ While this will not resolve networking problems, it will cause a process to fail
...
 +
 ----
 server.timeout =`number`
 ----
-. On the {productname} Proxy, open the [filename]``/etc/rhn/rhn.conf`` file and set a maximum timeout value (in seconds) for this parameter:
+. On the {productname} proxy, open the [filename]``/etc/rhn/rhn.conf`` file and set a maximum timeout value (in seconds):
```
Comment (keichwa) [inline modules/administration/pages/tshoot-rpctimeout.adoc]:
```suggestion
. On the {productname} Proxy, open the [filename]``/etc/rhn/rhn.conf`` file and set a maximum timeout value (in seconds):
```

File: modules/administration/pages/tshoot-rpctimeout.adoc (hunk 2 above)
Comment (keichwa) [reply modules/administration/pages/tshoot-rpctimeout.adoc]: another review req, pls

**Comment by keichwa** [comment]
thanks!

**Pull request merged**